code/modules/reagents/reagent_dispenser.dm

```diff
@@ -449,6 +460,9 @@
 		..()
 
 /obj/structure/reagent_dispensers/cauldron/barrel/attackby(obj/item/W, mob/user, params)
```

Comment:
Are you sure you aren't supposed to slap a ..() here

Reply:
🤷‍♂️

Reply:
It calls the parent. I'm not sure if it's strictly needed, though. I think generic actions like fingerprints are handled in `preattack`. But it wouldn't change anything to put one here.